src/ast.js

```diff
@@ -3363,5 +3378,115 @@ Sk.astDump = function (node) {
     return _format(node, "");
 };
 
+
+Sk.INHERITANCE_MAP = {
```

Comment:
what does this do is this automatically generated?

Reply:
Not automatically generated. I believe this data ended up here largely out of convenience that came when authoring it, and I'm thinking now it needs to be moved into the `src/lib/ast.js` file instead. The idea is that this encodes the inheritance relationships that exist between nodes. Want it moved to the library AST file?